torch/onnx/symbolic_helper.py

```diff
@@ -476,19 +476,43 @@ def _interpolate_warning(interpolate_mode):
                   "to support Pytorch's behavior (like coordinate_transformation_mode and nearest_mode).\n"
                   "We recommend using opset 11 and above for models using this operator.")
 
+
 def _unsqueeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Unsqueeze", input, axes)
-    else:
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Unsqueeze", input, axes)
         return g.op("Unsqueeze", input, axes_i=axes_i)
+    # Tensor type
+    if _export_onnx_opset_version < 13:
+        input_rank = _get_tensor_rank(input)
+        raise ValueError("Opset version must be >= 13 for Unsqueeze with dynamic axes. " +
+                         f"And accepted input axes range is [{-input_rank}, {input_rank-1}].")
+    return g.op("Unsqueeze", input, axes_i[0])
+
 
 def _squeeze_helper(g, input, axes_i):
-    if _export_onnx_opset_version >= 13:
-        axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
-        return g.op("Squeeze", input, axes)
-    else:
+    if _is_constant(axes_i[0]):
+        if _export_onnx_opset_version >= 13:
+            axes = g.op("Constant", value_t=torch.tensor(axes_i, dtype=torch.long))
+            return g.op("Squeeze", input, axes)
         return g.op("Squeeze", input, axes_i=axes_i)
+    # Tensor type
+    input_rank = _get_tensor_rank(input)
+    if _export_onnx_opset_version < 13:
+        print("_get_tensor_rank(axes_t):", input_rank)
+        raise ValueError("Opset version must be >= 13 for Squeeze with dynamic axes. " +
+                         f"And accepted input axes range is [{-input_rank}, {input_rank-1}].")
```

Comment:
I meant to include `input.node().sourceRange()` in the error message (if it's not empty)
On reflection we should probably add this universally for any symbolic function that fails, so don't add it here.
But for now just remove the second sentence here, go back to `"Opset version must be >= 13 for Squeeze with dynamic axes."`

Reply:
Added it. In my local test case, the input sourceRange is None, but it does not affect the error message show.